.github/workflows/ci.yml

```diff
@@ -78,3 +82,60 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: EmbarkStudios/cargo-deny-action@v1
+
+  convert-images:
```

Comment:
The job will slow down the CI run of each PR, can we run it only in daily testing (convet.yaml) ?

Reply:
Thanks! OK.

Reply:
Thanks! OK.